src/collection.js

```diff
@@ -239,6 +240,22 @@ Collection.prototype.onstart = function (evt) {
         if (self.actives.length < opts.maxNumberOfNipples) {
             self.processOnStart(touch);
         }
+        else if(origEvt.type.match(/^touch/)){
+            // zombies occur when end event is not received on Safari
+            // first touch removed before second touch, we need to catch up...
+            // so remove where touches in manager that no longer exist
+            Object.keys(self.manager.ids).forEach(function(k){
+                if(Object.values(origEvt.touches).findIndex(function(t){return t.identifier===k;}) < 0){
+                    // manager has id that doesn't exist in touches
+                    var e = [evt[0]];
+                    e.identifier = k;
+                    self.processOnEnd(e);
+                }
+            });
+            if(self.actives.length < opts.maxNumberOfNipples){
+                self.processOnStart(touch);
+            }
+        }
```

Comment:
I'll have to have a deeper look into this, I can't right now, but probably later today or in the week.